docs/develop/cmodules.rst

```diff
@@ -53,6 +53,30 @@ A MicroPython user C module is a directory with the following files:
   for header files), these should be added to ``CFLAGS_USERMOD`` for C code
   and to ``CXXFLAGS_USERMOD`` for C++ code.
 
+* ``usermod.cmake`` contains the CMake configuration for this module.
```

Comment:
What was the reason to call it `usermod.cmake`?  Why not `micropython.cmake` to match `micropython.mk`?

Reply:
I think the reasoning is lost to the sands of time, since I'm carrying this changeset over from the now ghost Raspberry Pi repository. This has actually been bugging me for a while and you're right- `micropython.cmake` is a lot clearer. Only reason I haven't changed it yet is because I have to rename a whole bunch of files across three repositories.

Reply:
I think it's worth changing now.  As you say, this PR sets a precedent.